setup.py

```diff
@@ -1,103 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+#
+# Note: To use the 'upload' functionality of this file, you must:
+#   $ pipenv install twine --dev
+
+import io
 import os
+import sys
+from shutil import rmtree
+
+from setuptools import find_packages, setup, Command
+
+# Package meta-data.
+NAME = "sbi"
+DESCRIPTION = "Simulation-based inference."
+URL = "https://github.com/mackelab/sbi"
+EMAIL = "sbi@mackelab.org"
+AUTHOR = "Álvaro Tejero-Cantero, Jakob H. Macke, Jan-Matthis Lückmann, Conor M. Durkan, Michael Deistler, Jan Bölts"
+REQUIRES_PYTHON = ">=3.7.0"
+
+REQUIRED = [
+    "joblib",
+    "matplotlib",
+    "numpy",
+    "pillow",
+    "pyro-ppl",
+    "pyknos==0.11",
+    "scipy",
+    "tensorboard",
+    "torch>=1.5.1",
+    "tqdm",
+]
+
+EXTRAS = {
+    "dev": [
+        "autoflake",
+        "black",
+        "deepdiff",
+        "flake8",
+        "isort",
+        "mkdocs",
+        "mkdocs-material",
+        "markdown-include",
+        "mkdocs-redirects",
+        "mkdocstrings",
+        "nbconvert",
+        "nbstripout",
+        "pep517",
+        "pytest",
+        "pyyaml",
+        "scikit-learn",
+        "torchtestcase",
+        "twine",
+    ],
+    "examples": ["ipywidgets", "jupyter", "notebook",],
+}
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+# Import the README and use it as the long-description.
```

Comment:
long description (no dash).

I wrote a long comment on why `desc` bla bla bla but actually the way we are careful with the README.md not to become a file readable by only developers at the moment is quite satisfactory, so I am in!